reason.opam

```diff
@@ -10,7 +10,7 @@ bug-reports: "https://github.com/facebook/reason/issues"
 dev-repo: "git://github.com/facebook/reason.git"
 tags: [ "syntax" ]
 build: [
-  ["jbuilder" "build"]
+  ["jbuilder" "build" "-p" name "-j" jobs]
```

Comment:
Sure. Suppose you have `reason-rtop` and `reason` opam packages in your repo. Here are some situations where your old build instructions are inadequate:

* A user doing `$ opam install reason` will build the code for `reason-rtop` as well. Which might fail because not all the deps might be installed by opam yet.

* A user doing `$ opam install reason-rtop` will be have rtop be built against the `reason` from its local source tree and not the one that is already installed by opam. This is wrong in cases of pins, users mixing and matching versions (which I don't suggest that you support. But you shouldn't break it out right).